packages/components/modules/payments/web/services/stripe.ts

```
@@ -0,0 +1,52 @@
+
+    if (allowIncomplete !== undefined) {
+      requestBody.allow_incomplete = allowIncomplete
+    }
+
+    if (billingDetails) {
+      requestBody.billing_details = billingDetails
+    }
+
+    return axios.post(`${baseUrl}/stripe/subscriptions`, requestBody)
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)